### PR TITLE
Typos/fixes to link syntax

### PR DESCRIPTION
**Pull request by Rocketknight1**
Noticed a couple of small errors and incorrect link syntax in the TPU tutorial, sorry about that!

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks for the fixes!

File: docs/source/en/perf_train_tpu_tf.mdx [hunk 1]
```diff
@@ -124,7 +124,7 @@ mean_loss = tf.reduce_sum(loss) / tf.reduce_sum(label_mask)
 
 Here, we avoid data-dependent shapes by computing the loss for every position, but zeroing out the masked positions in both the numerator and denominator when we calculate the mean, which yields exactly the same result as the first block while maintaining XLA compatibility. Note that we use the same trick as in rule #1 - converting a `tf.bool` to `tf.float32` and using it as an indicator variable. This is a really useful trick, so remember it if you need to convert your own code to XLA!
 
-**XLA Rule #3: XLA will need to recompile your model for every different input shape it sees**
+XLA Rule #3: XLA will need to recompile your model for every different input shape it sees**
```
Comment (sgugger) [inline docs/source/en/perf_train_tpu_tf.mdx:127]: No #### here?

**Review by sgugger — APPROVED** [review]
Thanks!

**Comment by Rocketknight1** [comment]
I was just checking what that formatting looked like before committing to it!

**Pull request merged**